pom.xml

```diff
@@ -263,14 +263,15 @@
     <repositories>
         <repository>
             <id>exist</id>
+            <!-- TODO(DP): fix the cert on the server and switch to https://repo.exist-db.org/repository/exist-db/ -->
             <url>http://repo.exist-db.org/repository/exist-db/</url>
```

Comment:
Why not just change this to the evolvedbinary URL? We have much better uptime than the place that is currently redirecting there